### PR TITLE
feat : added rating and grid-simplified components

### PR DESCRIPTION
**Pull request by Priya-Patel31**
Rating and grid-simplified component added.

### CODE REVIEW

File: component/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.rating-container{
+    display: flex;
+    gap:0.1rem;
+    font-size:1.5rem;
+}
+.checked{
+  color:gold;
```
Comment (t007rushi) [inline component/rating/rating.css]: You can include this color in the root

Comment (Priya-Patel31) [reply component/rating/rating.css]: done!!

File: component/rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.rating-container{
+    display: flex;
+    gap:0.1rem;
```
Comment (JantuDeb) [inline component/rating/rating.css:3]: Use px for very small length.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

File: component/rating/rating.css (hunk 2 above)
Comment (Priya-Patel31) [reply component/rating/rating.css:3]: I just want to be consistent throughout the code !!

**Review by t007rushi — APPROVED** [review]

**Pull request merged**